csharp/test/Ice/operations/MyDerivedClassAMDI.cs

```diff
@@ -9,9 +9,9 @@
 using System.Threading.Tasks;
 using Test;
 
-namespace ZeroC.Ice.Test.AMD.Operations
+namespace ZeroC.Ice.Test.Operations
 {
-    public sealed class MyDerivedClass : IMyDerivedClass
+    public sealed class MyDerivedClassAsync : IMyDerivedClassAsync
```

Comment:
I think we should rename the file too as `MyDerivedClassAsync .cs`

Reply:
What about the potential conflict with AMI files? I guess we'd use `-Async` for AMD implementations, and just leave `-AMI` for AMI implementations?

It seems a little asymmetric to me is all.
I'll blog about it, and maybe do it a separate PR anyways. If we switch from `-AMD` to `-Async`, there's a couple other places we'd want to as well I think that fall outside the scope of this one, like the actual test names too.

Reply:
Sure we can do that in a separate PR, I think the convention should be to name the file after the main class/type defined in it

Reply:
> Sure we can do that in a separate PR, I think the convention should be to name the file after the main class/type defined in it

I agree. Note that often the impl file name will the same as the generated file name (but in a different folder).